ONSdigital/florence
# Encrypt V4 on florence upload using vault

### What

Encrypt V4 on florence upload using vault

### How to review

- Ensure vault is running
- Run florence with encryption enabled
- Upload a v4File in florence, ensure the link displayed on upload is encrypted
- Run florence with encryption disabled
- Upload a v4File in florence, ensure the link displayed on upload is not encrypted

### Who can review

Anyone
